scripts/get_users_to_remind.py

```diff
@@ -0,0 +1,20 @@
+# The two inputs for this script are intentionally not shared
+# The users.py file is constant between runs and contains the hashes for all users
+# The forms.json file is exported directly from CosmosDB using the Data Migration Tool
```

Comment:
Maybe include the instructions on how to generate users.py and forms.json somewhere? Could be in a markdown file in github or a google doc linked from here.

Reply:
Yup it's in a private google doc that links to this script! Will share it with you / teammates